etc/profile-a-l/gnome-keyring-daemon.profile

```diff
@@ -0,0 +1,11 @@
+# Firejail profile for gnome-keyring-daemon
+# Description: Stores passwords and encryption keys
+# This file is overwritten after every install/update
+# Persistent local customizations
+include gnome-keyring-daemon.local
+# Persistent global definitions
+# added by included profile
+#include globals.local
+
+# Redirect
+include gnome-keyring.profile
```

Comment:
From gnome-keyring.profile:

> ```
> mkdir ${HOME}/.gnupg
> whitelist ${HOME}/.gnupg
> whitelist ${DOWNLOADS}
> ```

I assume that gnome-keyring-daemon is the Gnome equivalent of ssh-agent.

In that case, does the daemon itself really need direct access to the user's
keys?  Isn't there something analogous to ssh-add?

Reply:
If the access is not needed, it might be better to reverse the redirect:

* Make the base profile be gnome-keyring-daemon
* Make gnome-keyring allow the extra paths and redirect to gnome-keyring-daemon

Reply:
All valid points. Pushed the changes to reflect your review. Thanks!